lib/cocoapods/deintegrator.rb

```diff
@@ -23,6 +23,9 @@ def deintegrate_target(target)
         deintegrate_shell_script_phase(target, 'Check Pods Manifest.lock')
         deintegrate_shell_script_phase(target, 'Embed Pods Frameworks')
         deintegrate_pods_libraries(target)
+        target.build_configurations.each do |config|
+          config.base_configuration_reference.remove_from_project
```

Comment:
But I'm not sure what we can do if user manually added cocoapods config file inside her custom config file. Do we need a warning again?